WRITING_DOCS.md

```diff
@@ -9,8 +9,6 @@
   - [Divider](#divider)
```

Comment:
the 404 page is incorrectly themed now. It should be white, but it shows up a dark gray.
Making it impossible to read any text.
![image](https://github.com/icerpc/icerpc-docs/assets/25963603/e48c1f90-8672-4484-aa49-a878d41756a2)